deploy/charts/cert-manager/README.template.md

```diff
@@ -106,6 +106,13 @@ The following table lists the configurable parameters of the cert-manager chart
 | `affinity` | Node affinity for pod assignment | `{}` |
 | `tolerations` | Node tolerations for pod assignment | `[]` |
 | `topologySpreadConstraints` | Topology spread constraints for pod assignment | `[]` |
+| `livenessProbe.enabled` | Enable or disable the liveness probe for the controller container in the controller Pod. See https://cert-manager.io/docs/installation/best-practice/ to learn about when you might want to enable this livenss probe. | `false` |
+| `livenessProbe.initialDelaySeconds` | The liveness probe initial delay (in seconds) | `10` |
+| `livenessProbe.periodSeconds` | The liveness probe period (in seconds) | `10` |
+| `livenessProbe.timeoutSeconds` | The liveness probe timeout (in seconds) | `10` |
+| `livenessProbe.periodSeconds` | The liveness probe period (in seconds) | `10` |
+| `livenessProbe.successThreshold` | The liveness probe success threshold | `1` |
+| `livenessProbe.failureThreshold` | The liveness probe failure threshold | `8` |
```

Comment:
I added a new Helm chart value to allow the liveness probe to be enabled / disabled and configured.
Disabled by default, as discussed in https://kubernetes.slack.com/archives/CDEQJ0Q8M/p1682595470780429?thread_ts=1682499132.016199&cid=CDEQJ0Q8M